org.eclipse.winery.workflowmodeler/src/app/components/nodetemplate/node-template.component.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2017 ZTE Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and the Apache License 2.0 which both accompany this distribution,
+ * and are available at http://www.eclipse.org/legal/epl-v10.html
+ * and http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Contributors:
+ *     ZTE - initial API and implementation and/or initial documentation
+ */
+import { AfterViewInit, Component, Input  } from '@angular/core';
+import { Subscription } from 'rxjs/Subscription';
+import { WorkflowNode } from '../../model/workflow.node';
+import { BroadcastService } from '../../services/broadcast.service';
+import { WineryService } from '../../services/winery.service';
+import {Operation} from '../../model/operation';
+
+/**
+ * node template component provides operations about tosca modules which saved in winery.
+ * This component will be used in the property component while the corresponding workflow node is calling the node template's operation
+ */
+@Component({
+    selector: 'b4t-node-template',
+    templateUrl: 'node-template.component.html',
+})
+export class WmNodeTemplateComponent implements AfterViewInit {
+    @Input()
+    public node: WorkflowNode;
+    private nodeTemplates: any[] = [];

```

Comment:
Is `any` really required here? Isn't there an option to use an interface or class instead?